userspace/libsinsp/threadinfo.cpp

```diff
@@ -381,14 +388,13 @@ void sinsp_threadinfo::add_fd_from_scap(scap_fdinfo* fdi) {
 		tevt.set_info(&(g_infotables.m_event_info[PPME_SYSCALL_READ_X]));
 		tevt.set_cpuid(0);
 		tevt.set_num(0);
-		tevt.set_inspector(m_inspector);
```

Comment:
I removed the following line as the inspector is not anymore available in threadinfo. By inspecting the code, and by looking at the tests results, I couldn't find evidence this line is needed, but I'll be happier if someone can confirm that we actually don't need it.